src/platform_impl/linux/window.rs

```diff
@@ -911,6 +931,8 @@ pub enum WindowRequest {
   SetMenu((Option<menu::Menu>, AccelGroup, gtk::MenuBar)),
   GlobalHotKey(u16),
   SetVisibleOnAllWorkspaces(bool),
+  TaskbarProgressState(bool, Option<String>),
+  TaskbarProgress(f64, Option<String>),
```

Comment:
lets combine these two int one variant that takes `ProgressBarState` strcut
```suggestion
  PrgoressBarState(ProgressBarState),
```